packages/components/atoms/f-card/src/components/Card.vue

```diff
@@ -142,4 +148,13 @@ $card--pageContentWrapper-width           : 472px; // so that it falls on our 8p
     .c-card-heading--rightAligned {
         text-align: right;
     }
+
+    // The two card sizes used on the accounts page. e.g Previous orders & contact preferences cards.
+    .c-card--cardSizeCustom--large {
```

Comment:
Small naming thing – would just make this `c-card--sizeLarge` and `c-card--sizeMedium` as it's just a modifier on the base card class.